.github/workflows/wheels.yml

```diff
@@ -0,0 +1,107 @@
+name: Python wheels
+
+# TODO: use sscache to speed up this workflow
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    # fetch-depth: 0 is required to get the tags used to determine f3d version
```

Comment:
fetch depth is not used

Reply:
We'll need the tags in the future to build specific releases